templates/options.conf.erb

```diff
@@ -19,6 +19,15 @@ notify <%= scope.lookupvar('::dns::dns_notify') %>;
 <% unless [false, nil, :undefined, :undef, ''].include?(scope.lookupvar('::dns::listen_on_v6')) -%>
 listen-on-v6 { <%= scope.lookupvar('::dns::listen_on_v6') %>; };
 <% end -%>
+<% unless [false, nil, :undefined, :undef, ''].include?(scope.lookupvar('::dns::listen_on_53')) -%>
+listen-on port 53 { <%= scope.lookupvar('::dns::listen_on_53') %>; };
```

Comment:
`port 53` should be optional, best perhaps to leave it out of both the directive and the parameter name in case it was later parameterised, so it's generally just "listen_on".